SPDLOG_QUALITY_DECLARATION.md

```diff
@@ -92,15 +92,25 @@ Each of the `spdlog` source files containing code include a copyright statement
 
 ### Feature Testing [4.i]
 
-Many (if not all) of the advertised features of `spdlog` are tested as part of continuous integration.
+Each feature in spdlog has corresponding tests which simulate typical usage, and they are located in link to test source code. The result of these tests can be found at:
+ - [Linux](https://travis-ci.org/github/gabime/spdlog/builds/691497445)
+ - [macOS](https://travis-ci.org/github/gabime/spdlog/builds/691497445)
+ - [Windows](https://ci.appveyor.com/project/gabime/spdlog/builds/33133070)
+
+`spdlog` is pinned to a specific version by its vendor package, so we're fixed to a fixed set of features.
 
 ### Public API Testing [4.ii]
 
-There is some, but very little testing in `spdlog` specifically focused on API stability.
+`spdlog` has API tests which simulate typical usage, and they are located in link to test source code. The result of these tests can be found at:
+ - [Linux](https://travis-ci.org/github/gabime/spdlog/builds/691497445)
+ - [macOS](https://travis-ci.org/github/gabime/spdlog/builds/691497445)
+ - [Windows](https://ci.appveyor.com/project/gabime/spdlog/builds/33133070)
+
+`spdlog` is pinned to a specific version by its vendor package, so we're fixed to a fixed API.
 
 ### Coverage [4.iii]
 
-There is no test coverage tracking in `spdlog`.
+ROS 2 is using partially the `spdlog`'s API. Latest code coverage can be found [here](https://ci.ros2.org/job/ci_linux_coverage/lastSuccessfulBuild/cobertura/). The result shows the API that is being used in the ROS 2 codebase.
```

Comment:
```suggestion
ROS 2 is only using part of `spdlog`'s API. Latest code coverage can be found [here](https://ci.ros2.org/job/ci_linux_coverage/lastSuccessfulBuild/cobertura/). The result shows the API that is being used in the ROS 2 codebase.
```

I guess my other problem with this statement is that it is hard to see what part of the `spdlog` API is being used with the link given.  What exactly would a reader be looking for?

Reply:
The point here it's to explain that `spdlog` coverage is not complete in the coverage results because we are focus in the portion of API that we are using in the ROS 2 codebase. 

should we include the API functions that we are using in the ROS 2 codebase ?

Reply:
> should we include the API functions that we are using in the ROS 2 codebase ?

Yeah, I think that explaining which APIs we are using, and then explaining or showing coverage of just those features/APIs would make a lot more sense to me.

Reply:
something like this https://github.com/ros2/spdlog_vendor/pull/16/commits/8afd3d3ed0839f7cacf21024206c942d17b51041 or it's too little

Reply:
I would say that is a good start.  But for a reader of this QD, how can they tell that those APIs are being covered in the current tests?  We should have a link to either the upstream tests, or our tests, that makes it clear where those APIs are being exercised.